.github/workflows/ci.yml

```diff
@@ -45,9 +47,11 @@ jobs:
         submodules: 'true'
 
     - name: Set up JDK 11
-      uses: actions/setup-java@v1
+      uses: actions/setup-java@v3
       with:
         java-version: 11
+        distribution: 'zulu'
+        cache: 'sbt'
```

Comment:
The newer `setup-java` action requires an explicit JDK distribution. I set it to `zulu` since that's what it used implicitly, but I'm open to using a different distribution. (I honestly have no idea which distribution is the best here.)

Here's the list of supported distributions https://github.com/actions/setup-java?tab=readme-ov-file#supported-distributions